### PR TITLE
Update Extinction Curse Book 3 Ch1 NPCs

### PR DESCRIPTION
**Pull request by avagdumortesin**
Updated Bitter Truth Bandit, Pin Tingwheely, Raving Spirit, Swardlands Delinquent, and Tanessa Fleer.

### CODE REVIEW

File: packs/data/extinction-curse-bestiary.db/bitter-truth-bandit.json [hunk 1]
```diff
@@ -864,7 +866,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>A bandit deals an additional 2d6 precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                    "value": "<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/bitter-truth-bandit.json:869]:
```suggestion
                    "value": "<p>A bandit deals an additional 2d6 precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
```
Make sure you are leaving the actual book text there after you refresh an item.

File: packs/data/extinction-curse-bestiary.db/pin-tingwheely.json [hunk 2]
```diff
@@ -271,10 +271,9 @@
                     "value": true
                 },
                 "level": {
-                    "value": 2
+                    "value": 3
                 },
                 "location": {
-                    "heightenedLevel": 3,
                     "value": "aGUovHznkIyYG8m6"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/pin-tingwheely.json:277]: What happened here? Why did they heighten information vanish?

File: packs/data/extinction-curse-bestiary.db/pin-tingwheely.json [hunk 3]
```diff
@@ -1927,7 +1932,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p><strong>Trigger</strong> Pin is targeted with a melee or ranged attack by an attacker he can see.</p>\n<hr />\n<p>Pin gains a +2 circumstance bonus to AC against the triggering attack.</p>"
+                    "value": "<p><strong>Trigger</strong> A creature targets you with an attack and you can see the attacker.</p>\n<p><strong>Requirements</strong> You are not encumbered</p>\n<hr />\n<p>You deftly dodge out of the way, gaining a +2 circumstance bonus to AC against the triggering attack.</p>"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/pin-tingwheely.json:1935]:
```suggestion
                    "value": "<p><strong>Trigger</strong> Pin is targeted with a melee or ranged attack by an attacker he can see.</p>\n<hr />\n<p>Pin gains a +2 circumstance bonus to AC against the triggering attack.</p>"
```

File: packs/data/extinction-curse-bestiary.db/raving-spirit.json [hunk 4]
```diff
@@ -16,7 +16,7 @@
             },
             "stealth": {
                 "details": "<p>(expert) or DC 30 (master) to detect the spiritual nexus</p>",
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/raving-spirit.json:18]:
```suggestion
                "details": "<p>(expert) or @Check[type:perception|dc:30|trait:action:seek] (master) to detect the spiritual nexus</p>",
```

File: packs/data/extinction-curse-bestiary.db/swardlands-delinquent.json [hunk 5]
```diff
@@ -701,7 +704,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p><strong>Trigger</strong> The delinquent is targeted with a melee or ranged attack by an attacker she can see.</p>\n<hr />\n<p><strong>Effect</strong> The delinquent gains a +2 circumstance bonus to AC against the triggering attack.</p>"
+                    "value": "<p><strong>Trigger</strong> A creature targets you with an attack and you can see the attacker.</p>\n<p><strong>Requirements</strong> You are not encumbered</p>\n<hr />\n<p>You deftly dodge out of the way, gaining a +2 circumstance bonus to AC against the triggering attack.</p>"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/swardlands-delinquent.json:707]:
```suggestion
                    "value": "<p><strong>Trigger</strong> The delinquent is targeted with a melee or ranged attack by an attacker she can see.</p>\n<hr />\n<p><strong>Effect</strong> The delinquent gains a +2 circumstance bonus to AC against the triggering attack.</p>"
```

File: packs/data/extinction-curse-bestiary.db/swardlands-delinquent.json [hunk 6]
```diff
@@ -852,7 +855,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>The delinquent deals 1d6 damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                    "value": "<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/swardlands-delinquent.json:858]:
```suggestion
                    "value": "<p>The delinquent deals 1d6 damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
```

File: packs/data/extinction-curse-bestiary.db/tanessa-fleer.json [hunk 7]
```diff
@@ -700,15 +702,15 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>Tanessa can make Attacks of Opportunity with a loaded ranged weapon she's wielding if the triggering creature is within 5 feet of her.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.AttackOfOpportunity]</p>"
+                    "value": "<p>@Localize[PF2E.NPC.Abilities.Glossary.AttackOfOpportunity]</p>"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/tanessa-fleer.json:705]:
```suggestion
                    "value": "<p>Tanessa can make Attacks of Opportunity with a loaded ranged weapon she's wielding if the triggering creature is within 5 feet of her.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.AttackOfOpportunity]</p>"
```

**Review by SpartanCPA — CHANGES_REQUESTED** [review]

**Comment by SpartanCPA** [comment]
Pin Tingwheely's Spell Attack mod should be +18, not +16

**Comment by avagdumortesin** [comment]
I don't think I changed Pin's Spell Attack mod, but I'm not sure how the mod should be +18; shouldn't the mod be 10 less than the DC? His Primal Innate Spellcasting DC is 26 in the book, which would be +16 for the Spell Attack mod.

**Comment by TMunsie** [comment]
> I don't think I changed Pin's Spell Attack mod, but I'm not sure how the mod should be +18; shouldn't the mod be 10 less than the DC? His Primal Innate Spellcasting DC is 26 in the book, which would be +16 for the Spell Attack mod.

Unless stated explicitly, NPC’s are -8 for their spell attack, not -10. That’s a system convention because it is 90% what happens (the last 10% is -10)

File: packs/data/extinction-curse-bestiary.db/pin-tingwheely.json (hunk 2 above)
Comment (avagdumortesin) [reply packs/data/extinction-curse-bestiary.db/pin-tingwheely.json:277]: Let me look into this again.

Comment (avagdumortesin) [reply packs/data/extinction-curse-bestiary.db/pin-tingwheely.json:277]: Fixed with [5619553](https://github.com/foundryvtt/pf2e/pull/2726/commits/561955346d8ad8680ffb26469141e37bb70dcc81)

**Comment by avagdumortesin** [comment]
> Pin Tingwheely's Spell Attack mod should be +18, not +16

Fixed in [5619553](https://github.com/foundryvtt/pf2e/pull/2726/commits/561955346d8ad8680ffb26469141e37bb70dcc81)

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**